poset/poset_test.go

```diff
@@ -38,6 +38,8 @@ func TestPoset(t *testing.T) {
 			inputs[0].SetEvent(e)
 			assertar.NoError(
 				posets[0].ProcessEvent(e))
+			assertar.NoError(
```

Comment:
- add comments to the test (purpose of the test).
- add comment to function reorder()